syntax = "proto3";

service FileUploader {
  rpc UploadFile (stream FileChunk) returns (UploadStatus);
}

message FileChunk {
  string content = 1;
}

message UploadStatus {
  string message = 1;
}
